R/repository.R

```diff
@@ -1,32 +1,90 @@
-#' Create or open a git repository
+#' Create or discover a local git repository
 #'
-#' @description
-#' Use `git_init()` to start a new repository or `git_clone()` to download a
-#' repository from a remote.
-#'
-#' You may use `git_find()` and `git_open()` to explicitly discover and open
-#' existing git repositories, but this is usually not needed because all gert
-#' functions also take a path argument which implicitly opens the repo.
+#' Use `git_init()` to create a new repository or `git_find()` to discover an
+#' existing local repository. Each has some ability to recurse down or up, as
+#' appropriate.
```

Comment:
I did a fair amount of experimentation and reading libgit2 source to convince myself of what it does.